<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>工具集</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnMDMConf">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>270</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>基础数据配置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTool01">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>风力发电机组短路电流计算</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>系统配置</string>
    </property>
    <addaction name="actionConfigure"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>通用工具</string>
    </property>
    <addaction name="actionPowerCableCal"/>
    <addaction name="actionWireConvert"/>
    <addaction name="actionPowerConvert"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuTool"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="MenuMDM">
   <property name="text">
    <string>基础数据配置</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>基础数据配置</string>
   </property>
  </action>
  <action name="actionPowerCableCal">
   <property name="text">
    <string>动力电缆计算</string>
   </property>
  </action>
  <action name="actionWireConvert">
   <property name="text">
    <string>线束表转换</string>
   </property>
  </action>
  <action name="actionPowerConvert">
   <property name="text">
    <string>变压器计算</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>btnTool01</tabstop>
  <tabstop>btnMDMConf</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
